synthtool/gcp/templates/node_library/.github/auto-approve.yml

```diff
@@ -1,4 +1,2 @@
 processes:
-  - "NodeDependency"
-  - "OwlBotTemplateChangesNode"
-  - "OwlBotPRsNode"
\ No newline at end of file
+  - "NodeDependency"
```

Comment:
request for yaml not yml to be consistent

Reply:
I don't think I should change this one: https://github.com/googleapis/repo-automation-bots/blob/bc1194c1fe6f08803f64cb847809c8692a955854/packages/auto-approve/src/auto-approve.ts#L45

Reply:
Is it possible we could use .yml for the others above?

Reply:
+1 that's fine too